src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -56,142 +42,88 @@ public void Configure(AuthenticationOptions options)
                 return;
             }
 
-            // Register the OpenIddict handler in the authentication handlers collection.
-            options.AddScheme<OpenIddictServerHandler>(OpenIddictServerDefaults.AuthenticationScheme, displayName: null);
-
-            // If the userinfo endpoint was enabled, register a private JWT or validation handler instance.
-            // Unlike the instance registered by the validation feature, this one is only used for the
-            // OpenID Connect userinfo endpoint and thus only supports local opaque/JWT token validation.
-            if (settings.UserinfoEndpointPath.HasValue)
```

Comment:
In 3.0, the OpenIddict server services handle the access token validation process for userinfo requests, so we'll no longer have to register a separate validation/JWT handler instance.